main.go

```diff
@@ -114,12 +126,35 @@ func handleBadge(w http.ResponseWriter, r *http.Request) {
 	buf.WriteTo(w)
 }
 
+// save the cookies for any upstream calls to the Ory apis
+func withCookies(ctx context.Context, v string) context.Context {
+	return context.WithValue(ctx, "req.cookies", v)
```

Comment:
This is totally fine the way it's solved! If you want to be a Go-pro use static keys: https://stackoverflow.com/questions/40891345/fix-should-not-use-basic-type-string-as-key-in-context-withvalue-golint

Reply:
Good to know! 
Tbh this code was just copied from here https://github.com/ory/docs/blob/62c2a0a5d1936c2299baf4d26288bcd857cd2ce4/code-examples/protect-page-login/go/middleware.go#L15